packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -174,6 +174,16 @@ class RunCommand extends RunCommandBase {
   }
 
   @override
+  Future<Map<String, String>> get usageValues async {
+    final bool isEmulator = await devices[0].isLocalEmulator;
+    final String deviceName = devices.length == 1 
```

Comment:
`deviceType`? To make it clear that we're not sending the device name (which a user could have changed to arbitrary text).